modules/config.py

```diff
@@ -4,7 +4,7 @@
 CHANGELOG_MD_PATH   = "./changelog.md"
 CHANGELOG_REPO_PATH = "./data/"
 LOG_FILE_PATH       = "./LOG.log"
-GIT_TOKEN           = "GIT_TOKEN"  # Name of OS.ENV from which to pull the Github Token
+GIT_TOKEN           = []   # Name of OS.ENV from which to pull the Github Token
```

Comment:
I believe that we should keep the old string here (Eg `GIT_TOKEN = "GIT_TOKEN"`) as this tells people how they should name the `OS.Var`, as right now they won't know how to name the variables, and they will have to search for this line https://github.com/mozilla-releng/firefox-infra-changelog/blob/179b576fa3203e494c6136ee6b735041f82bc792/modules/FIC_Github.py#L35 

to figure out how to name their variables.
We then can store the available data in a property called `self.available_tokens = []  # Append all tokens in OS.Var here`